Lib/cu2qu/rf.py

```diff
@@ -48,29 +53,35 @@ def fonts_to_quadratic(*fonts, **kwargs):
     font at a time may yield slightly more optimized results.
     """
 
-    report = kwargs.get('report', {})
-    dump_report = kwargs.get('dump_report', False)
-    max_n = kwargs.get('max_n', 10)
+    if stats is None:
+        stats = {}
 
-    max_err_em = kwargs.get('max_err_em', 0.0025)
-    max_err = kwargs.get('max_err', None)
+    if max_err_em and max_err:
+        raise TypeError('Only one of max_err and max_err_em can be specified.')
+    if not (max_err_em or max_err):
+        max_err_em = DEFAULT_MAX_ERR
     if max_err:
         max_errors = [max_err] * len(fonts)
     else:
         max_errors = [f.info.unitsPerEm * max_err_em for f in fonts]
 
-    for glyph in FontCollection(fonts):
-        glyph_to_quadratic(glyph, max_errors, max_n, report)
+    if len(fonts) == 1:
```

Comment:
> oh sorry I misunderstood. Having a positional argument like "fontOrFonts" (ie. either a single or a list of fonts) would be kind of weird.

Ok, I see your point.  I'm fine with having two separate APIs.

>  I'm ok with using FontCollection all the time, even if length 1, provided it doesn't make things too slower.

With two separate APIs, the collection one never unwrapping, sounds about right.